includes/WebRequest.php

```diff
@@ -1134,6 +1134,23 @@ public function isValidWriteRequest( \User $user ) {
 	public function isWikiaInternalRequest() {
 		return $this->getHeader( self::WIKIA_INTERNAL_REQUEST_HEADER ) !== false;
 	}
+
```

Comment:
@garthwebb, we do backport changes from MW upstream. Such comments are really handy when solving conflicts.